<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pauseBtn">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>390</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minSpin">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QPushButton" name="resetBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="secSpin">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>361</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid black;
border-radius:10px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="randomASpin">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="randomBSpin">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="restTimeSpin">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="minLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>分钟:</string>
   </property>
  </widget>
  <widget class="QLabel" name="secLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>秒：</string>
   </property>
  </widget>
  <widget class="QLabel" name="randomALabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>随机范围a(m)</string>
   </property>
  </widget>
  <widget class="QLabel" name="randomBLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>随机范围b(m):</string>
   </property>
  </widget>
  <widget class="QLabel" name="restTimeLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>微休息时间(s)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back2homeBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;-back</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>pauseBtn</zorder>
  <zorder>minSpin</zorder>
  <zorder>resetBtn</zorder>
  <zorder>secSpin</zorder>
  <zorder>startBtn</zorder>
  <zorder>progressBar</zorder>
  <zorder>lcd</zorder>
  <zorder>randomASpin</zorder>
  <zorder>randomBSpin</zorder>
  <zorder>restTimeSpin</zorder>
  <zorder>minLabel</zorder>
  <zorder>secLabel</zorder>
  <zorder>randomALabel</zorder>
  <zorder>randomBLabel</zorder>
  <zorder>restTimeLabel</zorder>
  <zorder>back2homeBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
